src/layers/XRLayer/xr-layer-fragment.js

```diff
@@ -0,0 +1,27 @@
+export default `#version 300 es
```

Comment:
Would it be hard to have this without the JS wrapper? I guess we'd want a shader loader in webpack? Not a blocker, but please file an issue.